holoviews/plotting/bokeh/tabular.py

```diff
@@ -73,11 +73,12 @@ def current_handles(self):
         if self.static and not self.dynamic:
             return handles
 
+
+        element = self.current_frame
         previous_id = self.handles.get('previous_id', None)
-        current_id = id(self.current_frame.data) if self.current_frame else None
+        current_id = None if self.current_frame is None else id(element)+id(element.data)
```

Comment:
Summing ``ids`` is not a safe test for uniqueness as many pairs of ids could sum to the same value (even if it is unlikely!). Note that collisions might not even be *that* rare as the ``id`` doesn't return a hash - it is more like  to a memory address.

To check the identity of both the element and the data I would use the tuple ``(id(element), id(element.data))`` and not the sum.